yq.yaml

```diff
@@ -0,0 +1,27 @@
+package:
+  name: yq
+  version: 4.30.8
+  epoch: 0
+  description: "yq is a portable command-line YAML, JSON, XML, CSV and properties processor"
+  target-architecture:
+    - all
+  copyright:
+    - paths:
+        - "*"
+      attestation: TODO
+      license: Apache License 2.0
+  dependencies:
+    runtime:
+environment:
+  contents: {}
+  packages:
+    - busybox
+    - go
+pipeline:
+  - uses: go/install
+    with:
+      package: github.com/mikefarah/yq/v4
+      version: v${{package.version}}
+  - runs: |
+      ${{targets.destdir}}/usr/bin/yq --version | grep "version v${{package.version}}"
```

Comment:
testing?

Reply:
yeah, just want to make sure the right version of package is installed.

@tuananh ^^